recipes/natural_language_processing/rag/builds/Containerfile

```diff
@@ -1,4 +1,16 @@
 FROM registry.access.redhat.com/ubi9/python-311:1-52
```

Comment:
Why isn't this latest?

Reply:
we've pinned to this everywhere - @lmilbaum are you ok w/ switching to latest everywhere?

Reply:
My fear is that these do not age well.